extensions/cms/package.json

```diff
@@ -228,7 +228,8 @@
           "objectType": null,
           "categoryValues": null,
           "isRequired": false,
-          "isArray": false
+          "isArray": false,
```

Comment:
No, this will be updated to 3 category values once we are confirmed we want to support Strict. I will do that change separately in another PR. As of this PR, I only wanted to move the property over to main dialog.